test/unittests/LorisForms_Test.php

```diff
@@ -403,12 +408,14 @@ function testAddPageBreak()
      */
     function testAddElementSelect()
     {
-        $this->form = $this->getMockBuilder('LorisForm')
+        $form = $this->getMockBuilder('LorisForm')
```

Comment:
These need to be converted from a property to a simple variable for `@phan-var` to work on them. Adding the type hint isn't sufficient since phan picks up on the fact that it was assigned to a mock right before the call and uses that type instead of the phpdoc type, but you can't use `@phan-var` on a property..